spec/post_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author_id: 1, title: 'title', text: 'text', comments_counter: 0, likes_counter: 0) }
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should not exceed 250 characters' do
+    subject.title = 'a' * 255
+    expect(subject).to_not be_valid
+  end
```

Comment:
## * Kindly Add unit specs for all of your models' methods as unit tests for them are missing ( for example you have the method `recent_comments` ` update_user_posts_counter` which needs to be tested ) 
![image](https://github.com/torobucci/blog_app/assets/52497246/8955219d-0c4f-41dc-bf1a-ed6ade4df6db)